packages/razzle/config/createConfigAsync.js

```diff
@@ -806,6 +813,7 @@ module.exports = (
               })
             : null,
           new webpack.DefinePlugin(webpackOptions.definePluginOptions),
+          razzleOptions.writeCssDev ? miniCssExtractPlugin : null,
```

Comment:
razzleOptions.writeCssDev ||  IS_PROD ? miniCssExtractPlugin : null

Reply:
If I read the code correctly, I think this statement is already under IS_DEV.